agent/xds/clusters.go

```diff
@@ -772,6 +772,9 @@ func (s *ResourceGenerator) makeUpstreamClusterForPeerService(
 
 	clusterName := generatePeeredClusterName(uid, tbs)
 
+	outlierDetection := ToOutlierDetection(cfg.PassiveHealthCheck)
+	outlierDetection.MaxEjectionPercent = &wrappers.UInt32Value{Value: 100}
```

Comment:
Could we add a developer note explaining why?

If I were to trace this commit it would end up on this PR which doesn't have much context either